packages/host/public/config.js

```diff
@@ -1,4 +1,4 @@
 window.env = {
-  API_URL: 'http://localhost:4000',
-  // API_URL: 'http://localhost:8000/graphql',
+  // API_URL: 'http://localhost:4000',
+  API_URL: 'http://localhost:8000/graphql',
```

Comment:
I can update the version on demo-open.msupply.org and we could commit that URL rather than requiring a locally running server